platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Team.java

```diff
@@ -13,7 +14,7 @@
  */
 @Entity
 @Table(name = "team")
-public class Team extends IdentifiableEntity implements CanonicalEntity {
+public class Team extends IdentifiableEntity implements CanonicalEntity, Serializable {
```

Comment:
Class now implements `Serializable` but is missing field `serialVersionUID` which is strongly recommended.